Video.js

```diff
@@ -267,15 +267,15 @@ export default class Video extends Component {
         const getLicensePromise = Promise.resolve(getLicenseOverride); // Handles both scenarios, getLicenseOverride being a promise and not.
         getLicensePromise.then((result => {
           if (result !== undefined) {
-            NativeModules.VideoManager.setLicenseResult(result, findNodeHandle(this._root));
+            NativeModules.VideoManager.setLicenseResult(result,  data.contentId, findNodeHandle(this._root));
```

Comment:
Do this change impact client app ?

Reply:
It should not affect the client app because `setLicenseResult` and `setLicenseError` are not called by the client.